### PR TITLE
Users should be able to log in/log out

### PR DESCRIPTION
**Pull request by blakes24**
### What does this PR do?
- This PR allows users to log in and log out
### Description of Task to be completed
- A user enters their username and password to log in 
- A user clicks the logout link in the navbar to log out
### How should this be manually tested?
- Clone the repository
- Create an account, then click the logout link in the navbar
- Click the login link in the navbar and fill out the login form and submit

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -13,7 +13,8 @@
           "postgresql"
         ],
         "install": [
-          "pip install -r requirements.txt"
+          "pip install -r requirements.txt",
+          "pip install coveralls"
```
Comment (alexkayabula) [inline .travis.yml]: @blakes24 This is not part of the login feature. Make this a separate fix and create a separate pull request.

Comment (blakes24) [reply .travis.yml]: ok

**Comment by blakes24** [comment]
I removed the coveralls change.

**Comment by alexkayabula** [comment]
> I removed the coveralls change.

@blakes24 submit the fix pull request for the coveralls issue.

**Comment by blakes24** [comment]
I added a seed file. Try running that first to see if it fixes the server error on signup.

**Comment by alexkayabula** [comment]
LGTM
